setup.py

```diff
@@ -21,19 +21,20 @@
 DESC = 'Kriging Toolkit for Python'
 LDESC = 'PyKrige is a kriging toolkit for Python that supports two- and ' \
         'three-dimensional ordinary and universal kriging.'
-PACKAGES = ['pykrige']
+PACKAGES = ['pykrige', 'pykrige.optimise']
 PCKG_DAT = {'pykrige': ['README.md', 'CHANGELOG.md', 'LICENSE.txt', 'MANIFEST.in',
                         join('test_data', '*.txt'), join('test_data', '*.asc')]}
-REQ = ['numpy', 'scipy', 'matplotlib']
+REQ = ['numpy', 'scipy', 'matplotlib', 'sklearn']

```

Comment:
Fair enough, I can look into that. However, it's one of the most popular and better maintained packages. I see no reason to not include it.